src/function.py

```diff
@@ -93,14 +93,16 @@ def temporary_chdir(path: Union[str, Path]):
         os.chdir(old_path)
 
 
-def zip_and_upload_folder(client: CogniteClient, folder: str, name: str) -> int:
-    logger.info(f"Uploading code from '{folder}' to '{name}'")
+def zip_and_upload_folder(client: CogniteClient, code_directories: List[Path], name: str) -> int:
+    logger.info(f"Uploading code from {', '.join(map(str, code_directories))} to '{name}'")
     buf = io.BytesIO()  # TempDir, who needs that?! :rocket:
-    with temporary_chdir(folder), ZipFile(buf, mode="a") as zf:
-        for root_dir, _, files in os.walk("."):
-            zf.write(root_dir)
-            for f in files:
-                zf.write(Path(root_dir) / f)
+    with ZipFile(buf, mode="a") as zf:
+        for dir in code_directories:
+            with temporary_chdir(dir):
+                for root_dir, _, files in os.walk("."):
+                    zf.write(root_dir)
+                    for f in files:
+                        zf.write(Path(root_dir) / f)
```

Comment:
In that case `from common.lib import foo` will work in PyCharm/Pytest since PYTHONPATH is pointing to our repo root and in functions since you assemble similar relative structure.

Reply:
I'm not 100% sure here, but I suspect that won't pass integration test in deploy-functions

Reply:
Aha, I see, I'll change it up a bit. What I want is that the folder structure works for them locally

Reply:
I've changed to your suggested folder structure, i.e. the actual structure:
```
├── common
│   └── aasdfdsa.py
└── myfn
    └── handler.py
```
Is zipped to:
```
├── common
│   └── aasdfdsa.py
└── handler.py
```